v1/ao/internal/metrics/metrics.go

```diff
@@ -425,12 +424,44 @@ type MetricOptions struct {
 	Tags    map[string]string
 }
 
+func addRuntimeMetrics(bbuf *bson.Buffer, index *int) {
+	// category runtime
+	addMetricsValue(bbuf, index, "trace.go.runtime.NumGoroutine", runtime.NumGoroutine())
+	addMetricsValue(bbuf, index, "trace.go.runtime.NumCgoCall", runtime.NumCgoCall())
+
+	var mem runtime.MemStats
+	host.Mem(&mem)
+	// category gc
+	addMetricsValue(bbuf, index, "trace.go.gc.LastGC", int64(mem.LastGC)) // TODO may overflow
```

Comment:
Is this TODO still legit? :)

Reply:
ah technically it will but the overflow will happen only after the year 2264, so it should be fine. I'll remove this comment.